src/sql/parts/accountManagement/accountDialog/accountDialog.ts

```diff
@@ -199,10 +249,16 @@ export class AccountDialog extends Modal {
 
 		// Set the initial items of the list
 		providerView.updateList(newProvider.initialAccounts);
+
+		if (newProvider.initialAccounts.length > 0 && this._splitViewContainer.hidden) {
+			this.showSplitView();
+		}
+
 		this.layout();
 
-		// Store the view for the provider
+		// Store the view for the provider and action
 		this._providerViews[newProvider.addedProvider.id] = providerView;
```

Comment:
does it make more sense to have one map that points to an object that contains both `providerView` and `addAccountAction`?  If there's only ever two things indexed from provider id then two maps is probably fine, but it doesn't seem scalable if there could be more.